tesstrain/arguments.py

```diff
@@ -232,18 +243,18 @@ def show_tmpdir_location(training_dir: str) -> None:
     # specified in the command-line.
     if not ctx.training_text:
         ctx.training_text = (
-                pathlib.Path(ctx.langdata_dir) / ctx.lang_code / f"{ctx.lang_code}.training_text"
+            pathlib.Path(ctx.langdata_dir) / ctx.lang_code / f"{ctx.lang_code}.training_text"
```

Comment:
Please do not reformat code unnecessarily to keep the diff as small as possible.